docs/data-guide/big-data/non-relational-data-content.md

```diff
@@ -1,8 +1,14 @@
+A *non-relational database* is a database that does not use the tabular schema of rows and columns found in most traditional database systems. Instead, non-relational databases use a storage model that is optimized for the specific requirements of the type of data being stored. For example, data may be stored as simple key/value pairs, as JSON documents, or as a graph consisting of edges and vertices. Subsequently, there is no standardization which can lead to a learning curve for each type of NoSQL  database.
 
 What all of these data stores have in common is that they don't use a [relational model](../index.md). Also, they tend to be more specific in the type of data they support and how data can be queried. For example, time series data stores are optimized for queries over time-based sequences of data. However, graph data stores are optimized for exploring weighted relationships between entities. Neither format would generalize well to the task of managing transactional data.
 
-The term *NoSQL* refers to data stores that do not use SQL for queries. Instead, the data stores use other programming languages and constructs to query the data. In practice, "NoSQL" means "non-relational database," even though many of these databases do support SQL-compatible queries. However, the underlying query execution strategy is usually very different from the way a traditional relational database management system (RDBMS) would execute the same SQL query.
+The term *NoSQL* refers to data stores that do not use SQL for queries. Instead, the data stores use other programming languages and constructs to query the data. Subsequently, documentation, community resources and support varies for each NoSQL database. In practice, "NoSQL" means "non-relational database," even though many of these databases do support SQL-compatible queries. However, the underlying query execution strategy is usually very different from the way a traditional relational database management system (RDBMS) would execute the same SQL query.
+
+NoSQL databases offer more flexibility and scalability than SQL databases but at the cost of weaker consistency. You can read further about this [in here](https://azure.microsoft.com/en-us/resources/cloud-computing-dictionary/what-is-nosql-database) and [here](/dotnet/architecture/cloud-native/relational-vs-nosql-data). 
+
+> [!NOTE]  
+> Even though NoSQL databases allow you to store your data flexibly, you still need to identify and analyze your data access patterns, then design an appropriate data schema otherwise your NoSQL database can suffer under heavy workload.
+
```

Comment:
Please move these thoughts down to the end of the last section called "Typical requirements" and maybe make it something more like:

Compared to many traditional relational databases, NoSQL databases often offer a desirable level of schema flexibility and platform scalability but sometimes those benefits come at the cost of weaker consistency. Even though you can store your data flexibly, you still need to identify and analyze your data access patterns, then design an appropriate data schema otherwise your NoSQL database can suffer under heavy workload or unexpected usage patterns.

```suggestion
```